### ISSUE TITLE
Upgrader not opening Play store app

### ISSUE THREAD

**Issue by nabilfm** [larryaasen/upgrader#187]
Hi, I'm facing this case. When the alert's shown and click "update now", it doesn't open the play store app. Open in-app browser only. Tried on the android emulator & my phone.

I expect to open the Play store app

<img width="403" alt="image" src="https://user-images.githubusercontent.com/4882975/173978969-3ec653fe-f46a-4d66-92cb-3724795a0ba4.png">

```
I/flutter (17086): upgrader: instantiated.
I/flutter (17086): upgrader: build UpgradeAlert
I/flutter (17086): upgrader: languageCode: id
I/flutter (17086): upgrader: default operatingSystem: android VOG-xxxxx
I/flutter (17086): upgrader: operatingSystem: android
I/flutter (17086): upgrader: platform: TargetPlatform.android
I/flutter (17086): upgrader: isAndroid: true, isIOS: false, isLinux: false, isMacOS: false, isWindows: false, isFuchsia: false, isWeb: false
I/flutter (17086): upgrader: package info packageName: io.xxxxx
I/flutter (17086): upgrader: package info appName: xxxxx
I/flutter (17086): upgrader: package info version: 0.9.1
I/flutter (17086): upgrader: countryCode: ID
I/flutter (17086): upgrader: blocked: false
I/flutter (17086): upgrader: debugDisplayAlways: true
I/flutter (17086): upgrader: debugDisplayOnce: false
I/flutter (17086): upgrader: hasAlerted: false
I/flutter (17086): upgrader: shouldDisplayUpgrade: true
I/flutter (17086): upgrader: shouldDisplayReleaseNotes: true
I/flutter (17086): upgrader: showDialog title: Perbarui Aplikasi?
I/flutter (17086): upgrader: showDialog message: Versi terbaru dari xxx tersedia! Versi terbaru saat ini adalah 0.9.5 - versi anda saat ini adalah 0.9.1.
I/flutter (17086): upgrader: showDialog releaseNotes: bug fixes.
I/flutter (17086): upgrader: button tapped: update now
I/flutter (17086): upgrader: launching: https://play.google.com/store/apps/details?id=io.xxxxx
```
upgrader version number from the pubspec.lock file, which should look something like this:
```
  upgrader:
    dependency: "direct main"
    description:
      name: upgrader
      url: "https://pub.dartlang.org"
    source: hosted
    version: "4.2.1"
```

**Comment by nabilfm** [comment]
Sorry, didn't realize it's already submitted on this https://github.com/larryaasen/upgrader/issues/178

**Comment by larryaasen** [comment]
I just released the 4.2.2 update to the [ugprader package on pub.dev](https://pub.dev/packages/upgrader/versions/4.2.2) to fix this issue.
